src/components/ResultsPage.css

```diff
@@ -0,0 +1,59 @@
+a {
+  text-decoration: none;
+}
+
+.results-header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
```

Comment:
I could see that your header has a height of 160px. I think you can reduce the height to 100px so the content has a bit more room :)